ember-file-upload/addon/components/file-dropzone.ts

```diff
@@ -27,9 +31,20 @@ interface FileDropzoneArgs {
    *
    * Corresponds to `DataTransfer.dropEffect`.
    * (https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/dropEffect)
+   *
+   * @defaultValue 'copy'
    */
   cursor?: 'link' | 'none' | 'copy' | 'move';
 
+  /**
+   * Whether to add multiple files to the queue at once.
+   *
+   * If set to false only one file will be added when dropping mulitple files.
+   *
+   * @defaultValue true
+   */
+  multiple?: boolean;
```

Comment:
Decided not to deprecate as I can't think of an easy way to do this in user implementation.